File: TourApp/api/src/main/java/com/example/api/TourScrappingService.java
package com.example.api;

import java.util.List;

import com.example.api.model.TourStage;
import com.example.api.model.TourStageClasification;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

/**
 * Created by pedrofraca on 08/07/15.
 */
public interface TourScrappingService {
    @GET("/tour")
    Call<List<TourStage>> getStages();
    @GET("/clasification")
    Call<TourStageClasification> getClasificationForStage(@Query("stage") String stage);
}

File: TourApp/api/src/main/java/io/github/pedrofraca/tour/api/TourScrappingService.kt
package io.github.pedrofraca.tour.api

import io.github.pedrofraca.tour.api.model.Stage
import io.github.pedrofraca.tour.api.model.StageClassification
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path

interface TourScrappingService {
    @GET("/api/stage")
    fun stages(): Single<List<Stage?>>

    @GET("/api/stage/{stage}/classification")
    fun getClassificationForStage(@Path("stage") stage: String?): Single<StageClassification>
}